components/lib/sidebar/Sidebar.js

```diff
@@ -259,6 +264,9 @@ export const Sidebar = React.forwardRef((inProps, ref) => {
             {
                 className: 'p-sidebar-header'
             },
+            !props.header && {
+                style: { justifyContent: 'flex-end' }
```

Comment:
Instead of an inline style here just create a new CSS class like `p-sidebar-header` and then in this scenario just add another `p-sidebar-header-custom`

Reply:
I am not sure what ptm returns all it shows is `{}`. So it's not typed in a helpful manner. I haven't looked into it but I fixed it into a class and keeping it as mergeProps.